index.src.html

```diff
@@ -471,6 +496,8 @@ <h3 id="clear-response" algorithm>
             ::  <a lt="clear cookies" abstract-op>Clear cookies for |origin|</a>.
             :   "<a grammar>`storage`</a>"
             ::  <a lt="clear storage" abstract-op>Clear DOM-accessible storage for |origin|</a>.
+            :   "<a grammar>`clientHints`</a>"
+            ::  [=map/Set=] [=Accept-CH cache=][|origin|] to an empty [=ordered set=].
```

Comment:
Sets are lists, so you could use https://infra.spec.whatwg.org/#list-empty for a slightly more pithy instruction. That is:

```suggestion
            ::  [=list/Empty=] [=Accept-CH cache=][|origin|].
```